intranet/static/js/eighth/signupUI.js

```diff
@@ -185,98 +184,110 @@ $(function() {
         sh.scroll();
     });
 
-    var initX = null, initY = null, listening = false;
-    var $dayPicker = $(".day-picker");
-    $dayPicker.on("touchstart", function(e) {
-        e.stopPropagation();
-        initX = e.originalEvent.touches[0].clientX;
-        initY = e.originalEvent.touches[0].clientY;
-        listening = true;
-    });
-    $dayPicker.on("touchend", function(e) {
-        e.stopPropagation();
+    var initX = null,
+        initY = null,
         listening = false;
-    });
-    $dayPicker.on("touchmove", function(e) {
-        e.stopPropagation();
-        if (!listening) {
-            return;
-        }
-        var nowX = e.originalEvent.touches[0].clientX;
-        var nowY = e.originalEvent.touches[0].clientY;
-        if (Math.abs(nowY - initY) > 30) {
-            listening = false;
-            return;
-        }
-        var diffX = nowX - initX;
-        if (Math.abs(diffX) > 30) {
-            if (diffX > 0) {
-                $(".earlier-days").click();
-            }
-            else {
-                $(".later-days").click();
-            }
-            listening = false;
-        }
-    });
-    var activityPicker = $("#activityPicker");
-    activityPicker.on("touchstart", function(e) {
-        if ($("#activity-picker > .backbtn").hasClass("visible")) {
+
+    $(".day-picker").on({
+        "touchstart": function(e) {
             e.stopPropagation();
             initX = e.originalEvent.touches[0].clientX;
             initY = e.originalEvent.touches[0].clientY;
             listening = true;
-        }
-    });
-    activityPicker.on("touchend", function(e) {
-        if ($("#activity-picker > .backbtn").hasClass("visible")) {
+        },
+        "touchend": function(e) {
             e.stopPropagation();
             listening = false;
-        }
-    });
-    activityPicker.on("touchmove", function(e) {
-        var back = $("#activity-picker > .backbtn");
-        if (back.hasClass("visible")) {
+        },
+        "touchmove": function(e) {
             e.stopPropagation();
-            if (!listening) {
-                return;
-            }
-            var nowX = e.originalEvent.touches[0].clientX;
-            var nowY = e.originalEvent.touches[0].clientY;
+
+            if (!listening) return;
+
+            var nowX = e.originalEvent.touches[0].clientX,
+                nowY = e.originalEvent.touches[0].clientY;
+
             if (Math.abs(nowY - initY) > 30) {
                 listening = false;
                 return;
             }
+
             var diffX = nowX - initX;
+
             if (Math.abs(diffX) > 30) {
-                if (diffX > 0) {
-                    back.click();
-                }
+                if (diffX > 0) $(".earlier-days").click();
+                else $(".later-days").click();
+
                 listening = false;
             }
+        };
+    });
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/tjcsl/ion/file/2743148453/issues/source?bid=3176160&fileBranchId=3355598#l224)